central/scannerdefinitions/handler/updater.go

```diff
@@ -72,9 +80,15 @@ func (u *updater) runForever() {
 }
 
 func (u *updater) update() {
-	if err := u.doUpdate(); err != nil {
-		log.Errorf("Scanner vulnerability updater for endpoint %q failed: %v", u.downloadURL, err)
+	for attempt := 1; attempt <= u.RetryMax; attempt++ {
+		if err := u.doUpdate(); err != nil {
+			log.Warnf("Scanner vulnerability updater for endpoint %q failed attempt %d: %v", u.downloadURL, attempt, err)
+		} else {
+			return // successful update
+		}
+		time.Sleep(u.RetryDelay)
```

Comment:
I was wondering if we create a thundering herd with this retry, and my thoughts are inconclusive. 
We start the update process when we start the scanner, which is at unique times for all customers (today). 
There might be a situation where we publish a new ACS version, scanners auto-update at (around) the same time and request vulnerability update. 

Easy way to solve this would be to introduce some randomness/jitter to the retry.